vendor/bat-native-ledger/src/bat/ledger/internal/bitflyer/bitflyer_util.h

```diff
@@ -41,10 +41,6 @@ std::string GetAddUrl();
 
 std::string GetWithdrawUrl();
 
-type::ExternalWalletPtr GetWallet(LedgerImpl* ledger);
-
-bool SetWallet(LedgerImpl* ledger, type::ExternalWalletPtr wallet);
```

Comment:
`type::ExternalWalletPtr ResetWallet(type::ExternalWalletPtr wallet);` can be removed.